tests/test_server_functions.py

```diff
@@ -36,9 +37,9 @@ def validate_html(endpoint, document):
         )
 
 
-def test_root():
```

Comment:
Does this mean there is no test for the root anymore?

Reply:
It doesn't test root for html as it only redirects to `/library` in this new setup. But it is still tested for authentication in other test file.